xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -257,9 +257,11 @@ If this change is undesirable, you can configure your application to use the old
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the maximum number of allowed HTTP redirects.</summary>
+        <value>The maximum number of allowed HTTP redirects.</value>
+        <remarks>
+          <para />
```

Comment:
Remarks are optional, so they can be left with their original value:
```xml
<remarks>To be added.</remarks>
```
Same suggestion applies for the remarks below.